### DISCUSSION TITLE
Different icon color

### DISCUSSION THREAD

**Discussion by 9z0** [orgzly-revived/orgzly-android-revived#311] [General]
To be able to tell Orgzly from [Forgzly](8#discussioncomment-7652260). Including the icons for shortcut and sync under `src\main\res\`.

Thanks

**Comment by amberin** [comment]
Hm... What is the use case for having both apps permanenly installed? 🤔

**Reply by 9z0** [reply]
One for tasks and one for notes, each with different settings.

**Reply by 9z0** [reply]
That is a real use case for me.

Generally speaking, you can also have both when testing Revived. And a fork differentiating its icon is a good idea like [here](https://f-droid.org/en/packages/org.gdroid.gdroid/) or [here](https://f-droid.org/en/packages/io.github.quillpad)
